00-intro.md

```diff
@@ -0,0 +1,41 @@
+# 00. Develop Accessible Web Apps with React
+
+**[📹 Video](https://egghead.io/lessons/react-v2-00-a-beginners-guide-to-react-introduction?pl=a-beginners-guide-to-react-v2-6c4d)**
+
+**[💻 Course Repo](https://github.com/erin-doyle/egghead-react-a11y/tree/labels)**
+
+
+## Notes on Notes 🤓
+
+The word accessibility is often abbreviated to **a11y** with the number eleven in the middle (this refers to the number of letters that the word contains between the first and last letter - a numeronym).
+
+### What Is Web Accessibility?
+
+Web accessibility covers the following topics (and more):
+- keyboard navigation
+- focus
+- semantic HTML
+- color contrast
+- ARIA
+- alt attribute
+- error messages
+- captioning
+- instructions
+- readability
+- consistency
+
+### Important Acronyms
+
+- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/): Web Content Accessibility Guidelines
+- [WAI](https://www.w3.org/WAI/): Web Accessibility Initiative
+- [ARIA](https://www.w3.org/TR/wai-aria/): Accessible Rich Internet Applications
+
+### Emoji Legend
+
+| emoji| explanation              |
```

Comment:
Love the Emoji Legend! So helpful to have. This might be a cool thing to build up across notes 🙌

Reply:
Nice!! 😃